frontend/src/components/related/RelatedObject.tsx

```diff
@@ -15,41 +14,39 @@ type RelatedObjectRendererProps<T extends object> = {
  *
  * @param relatedObjects - The array of objects to render
  * @param view - The view type, either "DataGrid" or "AttributeGrid"
- * @param config - Tab configuration indicating view type and allowed fields
  */
 export function RelatedObjectRenderer<T extends object>({
-  object,
+  relatedObject,
   view,
   expandFields,
-  field,
 }: RelatedObjectRendererProps<T>) {
-  const relatedObjects = object._expand?.[field];
+  const _expandFields = ["transform", ...expandFields];
```

Comment:
Please make suret that the "transform" literal addition to expandFields originates from the same location as implementing or performing the transform, so that it's clear what's happening. I would move this up to overview.tsx so that it's visible in one place:

- What field is added to the related object.
- Waht field is used to render it.